package.json

```diff
@@ -9,13 +9,13 @@
 	"dependencies": {
 		"@preact/signals": "1.1.1",
 		"ethers": "6.6.4",
+		"funtypes": "5.0.3",
 		"preact": "10.8.1"
 	},
 	"scripts": {
-		"build": "tsc",
+		"build": "rm -rf app/js && tsc",
 		"serve": "npx serve -L ./app",
 		"vendor": "npm install && npm --prefix ./build install && npm --prefix ./build run vendor",
```

Comment:
instead of using "npm install" here, use "npm ci --ignore-scripts"

Reply:
updated the lock file and package.json with this strategy